cluster/images/kubemark/Makefile

```diff
@@ -15,8 +15,19 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
+REGISTRY?="gcr.io"
+PROJECT?="google_containers"
+
 all:
-	docker build --pull -t gcr.io/$(PROJECT)/kubemark .
-	gcloud docker -- push gcr.io/$(PROJECT)/kubemark
+	gcloudpush
+
+build:
+	docker build --pull -t $(REGISTRY)/$(PROJECT)/kubemark .
+
+gcloudpush: build
```

Comment:
I thought about this too, but it relies on DOCKER_PUSH_CMD by an external provider to be 'docker'. In my opinion, it's more readable in the external provider case to not have to run: `make DOCKER_PUSH_CMD=docker REGISTRY=192.xxx.xxx.xxx PROJECT=kubemark`.